### PR TITLE
Docs: (SEO): Adding meta description tags, minor editing to fix typos and improve some topic introductions

### PR DESCRIPTION
**Pull request by aaroncarey**
- Adds meta description tags and makes a few edits per SEO work and review by @moonstormer 
- Some edits to (or additions of) topic introductions as part of SEO
- Fixed a few typos

Fixes DGRAPH-2890

### CODE REVIEW

File: content/graphql/authorization/directive.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 weight = 2
 [menu.main]
     parent = "authorization"
 +++
 
-Given an authentication mechanism and a signed JSON Web Token (JWT), it's the `@auth` directive that tells Dgraph how to apply authorization.  The directive can be used on any type except `union` (that isn't a `@remote` type) and specifies the authorization for `query` as well as the `add`, `update`, and `delete` mutations. Additionally, this directive can also be used with the [`@secret`](/graphql/schema/types.md#password-type) directive. If you specify a `password` auth rule, Dgraph will use it to authorize the `check<Type>Password` query.
+Given an authentication mechanism and a signed JSON Web Token (JWT), the `@auth` directive tells Dgraph how to apply authorization.  The directive can be used on any type except `union` (that isn't a `@remote` type) and specifies the authorization for `query` as well as the `add`, `update`, and `delete` mutations. Additionally, this directive can also be used with the [`@secret`](/graphql/schema/types.md#password-type) directive. If you specify a `password` auth rule, Dgraph will use it to authorize the `check<Type>Password` query.
```
Comment (moonstormer) [inline content/graphql/authorization/directive.md:9]: Are authentication and authorization the same thing? Just want to make sure - I know we tend to use "auth" and I really appreciate spelling it out this way instead.

**Review by moonstormer — APPROVED** [review]
I had one question about auth, but looks great! Thanks so much for working through all of these!

File: content/graphql/authorization/directive.md (hunk 1 above)
Comment (aaroncarey) [reply content/graphql/authorization/directive.md:9]: "Auth" is overloaded (can be used to mean authentication or authorization), and best avoided in docs. JWTs are used for authentication (confirming identity / access to the DB) whereas the @auth directive is used for authorization (determining what an authenticated user can access).

**Review by bucanero — APPROVED** [review]
LGTM

**Pull request merged**